spec/features/user_sees_list_of_music_pieces_spec.rb

```diff
@@ -19,10 +23,10 @@
     end
```

Comment:
This test is doing too much, all you should do in this test is to make sure the titles for each piece is on the page. Thus, iterate over your `pieces` and for each one you should expect the page to have the title of the piece.